build.sbt

```diff
@@ -50,10 +50,29 @@ lazy val aggregatedProjects: Seq[ProjectReference] = Seq(
   discovery
 )
 
+/** TODO fix remoteTests in job conf and add remote coverage back.
+  * Does not include
+  * - planned removals in 2.6 https://github.com/akka/akka/milestone/119
+  * - docs?, protobuf, benchJmh
+  */
+lazy val aggregatedCoverage: Seq[ProjectReference] = Seq(
+  actor, actorTests,
+  cluster, clusterMetrics, clusterSharding, clusterTools, contrib,
+  discovery, distributedData, multiNodeTestkit, osgi,
+  persistence, persistenceQuery, persistenceShared, persistenceTck,
+  slf4j, stream, streamTestkit, streamTests, streamTestsTck, testkit,
+  actorTyped, actorTypedTests, actorTestkitTyped,
+  persistenceTyped, clusterTyped, clusterShardingTyped,
+  streamTyped)
```

Comment:
Would it be better as a derivate of the `aggregatedProjects` with projects removed instead? So that new modules in the future automagically get included.

Reply:
I thought about that, but what if a module is added that would need exclusion

Reply:
That's my only concern anyway, and why I did it this way. But now I'm leaning more towards the other.